titulos/routers_titulos.py

```diff
@@ -0,0 +1,27 @@
+from fastapi import APIRouter, Depends, Body, status, HTTPException
+from fastapi.responses import JSONResponse
+from sqlalchemy.orm import Session
+
+from titulos import crud, schemas
+from settings import bd_coneccion
+
+router = APIRouter()
+
+@router.post ("/Titulos", response_model = schemas.TituloRespuesta)
```

Comment:
Acordate que esto va en minusculas.

```suggestion
@router.post ("/titulos", response_model = schemas.TituloRespuesta)
```

Reply:
si se me paso